### PR TITLE
Feat: list for sale modal

### LINKED ISSUE #796: Implement the list for sale modal
## 🎉 Description

To complete this task:
- Implement the list for sale modal for the Asset Manage Page and remove the current list for sale implementation.

* [ ] 🐛 This is a bug report.
* [x] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by flobarreto**
image
image

closes #796

### CODE REVIEW

File: webapp/src/components/ManageAssetPage/Sell/Sell.container.ts [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
-const mapState = (_state: RootState): MapStateProps => ({})
+const mapState = (state: RootState): MapStateProps => {
+  return {
+    wallet: getWallet(state),
+    getContract: (query: Partial<Contract>) => getContract(state, query),
+    isCreatingOrder: isLoadingType(
+      getLoadingOrders(state),
+      CREATE_ORDER_REQUEST
+    )
+  }
```
Comment (LautaroPetaccio) [inline webapp/src/components/ManageAssetPage/Sell/Sell.container.ts:26]: This doesn't seem to be in used, would you mind removing it?

File: webapp/src/components/ManageAssetPage/Sell/Sell.tsx [hunk 2]
```diff
@@ -17,11 +15,12 @@ const Sell = (props: Props) => {
   const {
     className,
     rental,
-    order,
+    order = null,
```
Comment (LautaroPetaccio) [inline webapp/src/components/ManageAssetPage/Sell/Sell.tsx:18]: Would you mind explaining why is this variable set as null here?

File: webapp/src/components/Modals/SellModal/SellModal.tsx [hunk 3]
```diff
@@ -0,0 +1,359 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (LautaroPetaccio) [inline webapp/src/components/Modals/SellModal/SellModal.tsx:1]: Would you mind removing this rule?

File: webapp/src/components/Modals/SellModal/SellModal.tsx [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+  isCreatingOrder,
+  authorizations,
+  isAuthorizing,
+  error
+}: Props) => {
+  const { orderService } = VendorFactory.build(nft.vendor)
+
+  const [confirmedInput, setConfirmedInput] = useState<string>('')
+
+  const [stepper, setStepper] = useState(StepperValues.SELL_MODAL)
```
Comment (LautaroPetaccio) [inline webapp/src/components/Modals/SellModal/SellModal.tsx:57]: If I'm not wrong, this defines the `step` of the `stepper`, what do you think about calling it step?
```suggestion
  const [step, setStep] = useState(StepperValues.SELL_MODAL)
```

File: webapp/src/components/Modals/SellModal/SellModal.css [hunk 5]
```diff
@@ -0,0 +1,132 @@
+.fields-container {
```
Comment (LautaroPetaccio) [inline webapp/src/components/Modals/SellModal/SellModal.css:1]: Would you mind changing the styles file to use CSS modules? We're using modules for the new components.
Remember that you will need to use `:global` for the global classes.

File: webapp/src/components/Modals/SellModal/SellModal.css [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  }
+
+  .Authorization .ui.loader {
+    top: 8px;
+  }
+}
+
+.Authorization .ui.radio.checkbox .box:before,
+.Authorization .ui.radio.checkbox label:before {
+  width: 26px !important;
```
Comment (LautaroPetaccio) [inline webapp/src/components/Modals/SellModal/SellModal.css:123]: Is it possible to remove the importants by being more specific on the class definitions?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]
Looks good 👍 , it just needs a couple of changes.

File: webapp/src/components/ManageAssetPage/Sell/Sell.tsx (hunk 2 above)
Comment (flobarreto) [reply webapp/src/components/ManageAssetPage/Sell/Sell.tsx:18]: not needed

File: webapp/src/components/AuthorizationModal/AuthorizationModal.tsx [hunk 7]
```diff
@@ -43,19 +42,14 @@ const AuthorizationModal = (props: Props) => {
     address: authorization.contractAddress
   })
 
-  const hasFetchedAuthorizations = useRef(false)
+  // const hasFetchedAuthorizations = useRef(false)
```
Comment (LautaroPetaccio) [inline webapp/src/components/AuthorizationModal/AuthorizationModal.tsx:45]: Would you mind removing this comment?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**